### PR TITLE
Rls 0.6.0

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @TomAugspurger. Couple suggestions:

1. Should switch to [using `pip`]( https://conda-forge.org/docs/meta.html#use-pip ) to build and install.
2. Should update how we [build against `numpy`]( https://conda-forge.org/docs/meta.html#building-against-numpy ).

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,23 +13,24 @@ source:
...
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
-    - toolchain
+    - pip
+    - numpy 1.9.*     # [not (win and py36)]
+    - numpy 1.11.*    # [win and py36]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Not sure how important `numpy` version `1.9` support is for us here. If not at all important, we can change this to just `numpy 1.11.*` (without selectors) or similar.

**Comment by jakirkham** [comment]
Pushed a few changes to your PR to fix the build. Hope that is ok. Also please take a look and let me know if there are any questions.

**Comment by TomAugspurger** [comment]
Thanks, I appreciate it.

I just did numpy 1.9 out of habit. Happy to bump it to 1.11.

________________________________

**Comment by jakirkham** [comment]
Thanks :)

**Comment by jakirkham** [comment]
All packages are now published. 🎉

**Comment by TomAugspurger** [comment]
Thanks!

**Pull request merged**